app/_locales/es/messages.json

```diff
@@ -929,5 +929,579 @@
   },
   "yourSigRequested": {
     "message": "Tu firma ya fue solicitada"
+  },
+  "contractInteraction": {
+    "message": "Interacción del Contrato"
+  },
+  "providerRequest": {
+    "message": "$1 quisiera conectar con tu cuenta"
+  },
+  "acceleratingATransaction": {
+    "message": "* Agilizar a una transacción al usar un precio de gas más alto aumenta las probabilidades que que la red lo procese más rápidamente, pero eso no siempre está garantizado."
+  },
+  "accessingYourCamera": {
+    "message": "Accediendo a tu cámara..."
+  },
+  "accountOptions": {
+    "message": "Opciones de la Cuenta"
+  },
+  "accountSelectionRequired": {
+    "message": "¡Necesitas elegir una cuenta!"
+  },
+  "activityLog": {
+    "message": "registro de actividades"
+  },
+  "advancedOptions": {
+    "message": "Opciones Avanzadas"
+  },
+  "addSuggestedTokens": {
+    "message": "Agregar tokens propuestos"
+  },
+  "addAcquiredTokens": {
+    "message": "Agregar los tokens que has adquirido usando MetaMask"
+  },
+  "advanced": {
+    "message": "Avanzado"
+  },
+  "amountPlusTxFee": {
+    "message": "Cantidad + Comisión por transacción"
+  },
+  "attemptToCancel": {
+    "message": "¿Intentar cancelar?"
+  },
+  "attemptToCancelDescription": {
+    "message": "El intentar cancelar la transacción no garantiza la cancelación de la misma. Si el intento de cancelación tiene éxito, se le cobrará la comisión de transacción indicada arriba."
+  },
+  "average": {
+    "message": "promedio"
```

Comment:
This doesn't start with a capital letter?

Reply:
Correcto. Lo cambio.